node/src/Pos/Lrc/Worker.hs

```diff
@@ -197,7 +197,7 @@ lrcDo epoch consumers tip = tip <$ do
     whileAfterCrucial b = getEpochOrSlot b > crucial
     crucial = EpochOrSlot $ Right $ crucialSlot epoch
     withBlocksRolledBack blunds =
-        bracket_ (rollbackBlocksUnsafe blunds)
+        bracket_ (rollbackBlocksUnsafe (BypassSecurityCheck True) blunds)
```

Comment:
Because LRC can rollback more than `k` blocks :man_shrugging: 

I don't know why it can do it, so my comment would be hardly of any help. If someone more knowledgeable added a comment, it'd be great.